cdap-docs/developer-manual/source/pipelines/creating-pipelines.rst

```diff
@@ -212,6 +212,94 @@ Two interfaces are available:
   underlying ``cron`` program. The details of that program will depend on the operating
   system used by the host of the CDAP Master process.
 
+With the CDAP Studio, you can also create a schedule that launches a pipeline when another pipeline
+when another pipeline is completed, failed, or killed. Arguments and plugin properties from the triggering pipeline
+can be passed to the triggered pipeline as arguments.
+
+:ref:`HTTP request <http-restful-api-lifecycle-schedule-add>` can also be used to create such schedule.
+Arguments and plugin properties from the triggering pipeline can be passed to the triggered pipeline by setting
+a special schedule property. Set the 'triggering.properties.mapping' property to the stringified value of
+the following JSON object::
+
+    {
+      "arguments" : [
+        {
+          "source" : <runtime argument key in the triggering pipeline>,
+          "target" : <runtime argument key in the current pipeline>
+        },
+        ...
+      ],
+      "pluginProperties" : [
+        {
+          "stageName" : <stage name in the triggering pipeline>,
+          "source" : <plugin property key in the given stage in the triggering pipeline>,
+          "target" : <runtime argument key in the current pipeline>
+        },
+        ...
+      ]
+    }
+
+For example, suppose you have pipeline 'X' in namespace 'nsX' and pipeline 'Y' in namespace 'nsY'.
+Pipeline 'X' is configured to write to a table set by the 'output' argument, and pipeline 'Y' is configured to read
+from a table set by the 'input' argument. You want to create a schedule so that pipeline 'Y' is run whenever
+pipeline 'X' completes, with pipeline 'Y' configured to read from the output table of pipeline 'X'. In other words,
+if pipeline 'X' completes with argument 'output'='xyTable', you want pipeline 'Y' to run with argument
+'input'='xyTable'. To do this, the 'triggering.properties.mapping' schedule property should contain
+the 'arguments' field::
+
+    "arguments" : [
+      {
+        "source" : "output",
```

Comment:
be consistent with spaces. remove the space before ':' here, and add a space after ':' in the line below